### ISSUE TITLE
Logo scrolling accessibility

### ISSUE THREAD

**Issue by murrayinman** [styled-components/styled-components-website#769]
Greetings! 
I wanted to suggest a change to the [logo scrolling sections of the index.js](https://github.com/styled-components/styled-components-website/blob/4afb1d11f482c752768228f784bafcdce77035b9/pages/index.js#L215) page. While investigating this tool, I saw the auto-scrolling section for the logos for the companies using Styled Components. I didn't keep the section in the viewport for very long because it quickly made me ill. Over an hour later, my head hurts and I feel the sickening vertigo. 

I rarely have issues with motion and thus rather not set my OS to prefers-reduced-motion` which could remove all animated experiences. I told chrome to emulate the `prefers-reduced-motion` CSS media feature and the page respected it and stopped the scrolling. (WIN!) However, it still fails [WCAG 2.1 Criterion 2.2.2: Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html). 

I suggest adding a way for the user to stop the scrolling action. This will have the additional benefit to users that need extra time to understand what the logo is before it scrolls away.  

If nothing else, slowing the animation down to 120s or more would really help.

_Thanks!_
Murray

**Comment by erickrod54** [comment]
Hello Mr @murrayinman, the issue to make **reduced-motion**  can be made modifying:

`const CompanyLogo = styled.span`
  position: relative;
  height: ${(p) => p.height || '2rem'};
  margin: 0 1rem;
  bottom: ${(p) => p.bottom || 0};
  opacity: 0.8;
  filter: brightness(0) invert(.5); /*this prop that currentlly is set as '1' **/
  transition: opacity 125ms ease-in-out;`

I think also the transition can be set lower, in terms of accesibility is a very important matter for the motion-sickness condition as you pointed :)
